packages/nvidia-container-toolkit/nvidia-container-toolkit.spec

```diff
@@ -20,7 +20,6 @@ Source3: nvidia-oci-hooks-json
 BuildRequires: %{_cross_os}glibc-devel
 Requires: %{_cross_os}libnvidia-container
 Requires: %{_cross_os}shimpei
-Requires: %{_cross_os}nvidia-k8s-device-plugin
```

Comment:
We're removing this dependency but should it be reversed? Does `nvidia-k8s-device-plugin` work as expected if `nvidia-container-toolkit` is not installed?

Reply:
Yes, `nvidia-k8s-device-plugin` doesn't require `nvidia-container-toolkit` to work properly, it won't try to call any binary provided by `nvidia-container-toolkit` and it only registers itself with the kubelet.